FABRIC-AND-CFT.md

```diff
@@ -4,85 +4,85 @@ This page highlights the main differences (both technical and philosophical) bet
    </td>
   </tr>
   <tr>
    <td><strong>Configuration</strong>
    </td>
-   <td>Less opinionated allowing end users higher flexibility.
+   <td>Less opinionated, allowing end users higher flexibility and often offering more configuration options within a single module.
    </td>
-   <td>Opinionated by default, end users may need to fork if it does not meet their use case.
+   <td>Opinionated by default, end users may need to fork if it doesn't fully meet their use case.  Often requires composing multiple modules for complex configurations.
    </td>
   </tr>
   <tr>
    <td><strong>Extensibility</strong>
    </td>
-   <td>Built with extensibility in mind catering to fork and use patterns. Modules are often lightweight and easy to adopt / tailor to specific use cases.
+   <td>Built with extensibility in mind, catering to fork and use patterns. Modules are often lightweight and easy to adopt / tailor to specific use cases.
    </td>
-   <td>Not built with fork and use extensibility catering primarily to off the shelf consumption. Modules are tailored towards common usecases and extensible via composition.
+   <td>Not primarily built for fork-and-use extensibility, catering primarily to off-the-shelf consumption. Modules are tailored towards common use cases and extensible via composition.
    </td>
   </tr>
   <tr>
-   <td><strong>Config customization</strong>
+   <td><strong>Config Customization</strong>
    </td>
-   <td>Prefer customization using variables via objects, tight variable space.
+   <td>Prefer customization using variables via objects, providing a tight but comprehensive variable space within modules.
    </td>
-   <td>Prefer customization using variables via primitives.
+   <td>Prefer customization using variables via primitives, which can lead to more modularity but may require using multiple modules for customization.
    </td>
   </tr>
   <tr>
    <td><strong>Examples</strong>
    </td>
-   <td>Thorough examples for individual modules, and end to end examples composing multiple modules covering a wide variety of use cases from foundations to solutions.
+   <td>Thorough examples for individual modules and end-to-end examples composing multiple modules, covering a wide variety of use cases from foundations to solutions. 
    </td>
    <td>Examples for a module mostly focus on that individual module. Composition is often not shown in examples but in larger modules built using smaller modules.
    </td>
   </tr>
   <tr>
    <td><strong>Resources</strong>
    </td>
-   <td>Leaner modules wrapping resources.
+   <td>Modules often encapsulate a broader set of resources, providing more functionality within a single unit.
    </td>
-   <td>Heavier root modules that often compose leaner sub modules wrapping resources.
+   <td>Heavier root modules often compose leaner sub-modules wrapping resources, which can lead to more granular control but potentially increased complexity.
```

Comment:
what does "granular control" mean? What can you do with that control that Fabric doesn't let you do?

Reply:
Good point. "Granular control" needs to be more specific.

Rather than CFT inherently offering more control, it's about how that control is distributed. Fabric often provides a broader set of configuration options within a single module, while CFT might break those options into separate modules.

Take the same examples from our other discussion thread:

Google Storage: Fabric's `gcs` module handles IAM, tag bindings, etc., all in one place. CFT would require additional modules or resources for those aspects.
Project Setup: Fabric's `project` module covers IAM permissions, custom roles, and more. CFT's `project-factory` might handle some of those features through separate modules.

So, it's not necessarily about more control, but rather where that control is located. Fabric leans towards comprehensive modules, which can streamline configurations, while CFT's modularity might offer more flexibility in certain cases.